shuup/admin/template_helpers/shuup_admin.py

```diff
@@ -44,6 +44,17 @@ def get_menu_entry_categories(context):
     return menu.get_menu_entry_categories(request=context["request"])
 
 
+def is_menu_category_active(category, target_url):
+    def does_identifier_match(always_active_identifier):
+        return bool("%s" % category.identifier == "%s" % always_active_identifier)
+
+    identifiers = settings.ALWAYS_ACTIVE_MENU_CATEGORY_IDENTIFIERS
+    if any(identifier for identifier in identifiers if does_identifier_match(identifier)):
+        return True
+
+    return any([entry for entry in category.entries if entry.url == target_url])
```

Comment:
you need to be consistent. you don't use brackets on the `any` above but now you use. pick up one :)